plugins/nf-co2footprint/src/resources/CO2FootprintReportTemplate.html

```diff
@@ -199,6 +200,43 @@ <h4>Workflow execution completed unsuccessfully!</h4>
         <dt class="col-sm-3">Nextflow version</dt>
         <dd class="col-sm-9">version ${workflow.nextflow.version}, build ${workflow.nextflow.build} (${workflow.nextflow.timestamp})</dd>
       </dl>
+
+      <dt>Total CO2 footprint measures</dt>
+      <dl class="row small">
+        <dt class="col-sm-3">Total CO2 emission</dt>
+        <dd class="col-sm-9">${co2_totals.co2} g</dd>
+
+        <dt class="col-sm-3">Total energy consumption</dt>
+        <dd class="col-sm-9">${co2_totals.energy} Wh</dd>
+      </dl>
+
+      <p>This is equivalent to:</p>
+      <dl class="row small">
+        <dt class="col-sm-6"><i class="fa fa-tree"></i> Months a tree would need to sequester this amount of CO2</dt>
+        <dd class="col-sm-3">${co2_totals.tree}</dd>
+
+        <dt class="col-sm-6"><i class="fa fa-car"></i> Km by car</dt>
+        <dd class="col-sm-3">${co2_totals.car}</dd>
+
+        <dt class="col-sm-6"><i class="fa fa-plane"></i> Km by plane</dt>
```

Comment:
I would tend to stick to only one plane equivalence value.

Reply:
Sounds good, should we keep the % of a flight Paris-London to have a different metrics? As we already have km by car